config/scale/cluster/samples/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+namespace: ibm-spectrum-scale
+
+commonLabels:
+  app.kubernetes.io/instance: ibm-spectrum-scale
+  app.kubernetes.io/name: cluster
+
+resources:
+  - ../bases
+  - ../cluster
+
+patchesStrategicMerge:
+  - delete_filesystem.yaml
```

Comment:
need to delete this line, looks like we removed the vdisknsd-sample, kustomize build fails here.

Reply:
Thanks for addressing this!